cerbero/build/build.py

```diff
@@ -221,6 +221,7 @@ class Autotools (MakefilesBase):
     add_host_build_target = True
     can_use_configure_cache = True
     supports_cache_variables = True
+    enable_gtkdoc = False
```

Comment:
Instead of creating an enable_gtkdoc variable, better do it in two steps:
* Define a variable like you did named use_gtkdoc
* Define a new variant called 'docs', similar to 'debug' or 'static', etc.